### DISCUSSION TITLE
一般的なWebサイト/アプリに必要な、規約ページについて

### DISCUSSION THREAD

**Discussion by kazzyfrog** [lgtm-factory/lgtm-factory#162] [Scrap]
一般的なWebサイト/アプリに必要な、規約ページについて、つらつら書いていきますー！

これ記事かけたら書きたい！
何度も使いまわせそうなことなので！

**Comment by kazzyfrog** [comment]
- プライバシーポリシー
- 免責事項
- お問い合わせ
- 利用規約
- （決算ある場合は、特商標）

**Reply by kazzyfrog** [reply]
今回は、利用規約を作成する。
そこに、免責事項の記載する！

また、統計・アカウント認証を入れる場合は、プライバシーポリシーで、個人情報の取り扱いについて書いたほうが良い

**Comment by kazzyfrog** [comment]
お問い合わせフォームは、正直、専用メアドか、Googleフォームで良い！
特に初期は！

**Reply by kazzyfrog** [reply]
ただ、これも一応Googleから推奨されている！
しっかりしたサイトなら、おいてね的な。

**Reply by kagomen** [reply]
自分もGoogleフォームで良いと思います🐈
あと、前にアストロでブログ作った時に利用した https://ssgform.com/ が、簡単で良い感じだったので共有！

**Reply by kagomen** [reply]
あ、月100件までなんだ👀

**Reply by kazzyfrog** [reply]
SSGform 知りませんでした！！
Next.jsと、相性良さそうですね🤞
